joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -299,3 +386,28 @@ def update(self, delta):
                 if joint['position'] < joint['min']:
                     joint['position'] = joint['min']
                     joint['forward'] = not forward
+
+
+def main(input_args=None):
```

Comment:
AFAIK, the `console_scripts` entry point doesn't call the function with any arguments, so `input_args` can be deleted.

https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point

Reply:
All right.  The input_args thing is a pattern I've seen elsewhere in the codebase, but I don't feel strongly about it.  Fixed in 2917803.